walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -13,11 +13,111 @@
 // limitations under the License.
 
 package com.google.sps;
-
 import java.util.Collection;
+import java.util.ArrayList;
+import java.util.Set;
+import java.util.HashSet;
 
 public final class FindMeetingQuery {
+  private static final int TIME_INCREMENT = 1;
+
+  /**
```

Comment:
For function comments, Google tends to follow the Java Doc style. https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html

Specifically the @returns and @params tags